sage-ios/SAGE/SAGE/Announcement.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Announcement.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 10/3/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class Announcement: NSObject {
+    
+    var id: Int?
+    var sender: User?
+    var title: String?
+    var text: String?
+    var timeCreated: NSDate?
+    var school: School?
+    
+    init(id: Int, sender: User, title: String, text: String, timeCreated: NSDate, school: School) {
```

Comment:
try setting all of these parameters as optionals with default values (equivalent to 0 or nil), so that we can just pass any paramters we want during init. This way, we can just do something like Announcement(title: "s", text: "hello") instead of cluttering the init with mostly nil values